blog.html

```diff
@@ -0,0 +1,130 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>Sprint Challenge - Blog</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <a href=index.html><img class="logo" src="https://i.imgur.com/tlyNooH.png">
+                <nav>
+                    <a href=index.html>Home</a>
+                    <a href="about.html">About</a>
+                    <a href="#">Products</a>
+                    <a href="#">Blog</a>
+                    <a href="contact.html">Contact</a>
+                </nav>
+        </header>
+        <img class="blog-image" src="https://i.imgur.com/mqkHY0G.jpg">
+        <div class="blog-content">
+            <section class="top-content1">
+                <h4>Lambda Blog</h4>
+                <p class="introduction">This blog was created to discuss the learning experiences of all of Lambda
+                    school's cohorts and their
```

Comment:
I love the blog!  It's so creative!  It's cool how the hover flips the text in different directions.